core/metacling/src/TClingValue.cxx

```diff
@@ -44,6 +44,25 @@ TClingValue& TClingValue::operator=(TClingValue &Other) {
    return *this;
 }
 
+std::pair<std::string, std::string> TClingValue::ToTypeAndValueString() const {
+  std::string output = ToString();
+  int pos, bracket = 0, str_size = output.size() - 1;
+  for (pos = 0; pos <= str_size; pos++) {
+    if (output[pos] == '(')
+      bracket++;
```

Comment:
"bracket" is `[`. You mean "paren". And you use this to count the nesting level. So maybe just `nest`? Or `paren_level`?